modules/fundamental/src/vulnerability/service/test.rs

```diff
@@ -60,3 +60,29 @@ async fn statuses(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+#[test_context(TrustifyContext, skip_teardown)]
+#[test(actix_web::test)]
+async fn status_duplicates(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
+    let db = ctx.db;
+    let vuln_service = VulnerabilityService::new(db.clone());
+    let (storage, _tmp) = FileSystemBackend::for_test().await?;
+
+    let ingestor = IngestorService::new(Graph::new(db.clone()), storage);
+
+    // ingest an advisory
+    let data = include_bytes!("../../../../../etc/test-data/csaf/CVE-2024-5154.json");
+    let data = ReaderStream::new(&data[..]);
+
+    ingestor
+        .ingest(("source", "test"), None, Format::CSAF, data)
+        .await?;
+
+    let vuln = vuln_service
+        .fetch_vulnerability("CVE-2024-5154", Transactional::None)
+        .await?;
+
+    println!("{}", serde_json::to_string_pretty(&vuln)?);
```

Comment:
Can you assert instead of print?

Reply:
It ended up being a duplicate test. Rm'd.